cleanlab_studio/studio/upload.py

```diff
@@ -17,16 +17,20 @@ def upload_dataset(
     schema_overrides: Optional[FieldSchemaDict] = None,
     modality: Optional[str] = None,
     id_column: Optional[str] = None,
-):
+) -> str:
     upload_id = upload_dataset_file(api_key, dataset_source)
     schema = get_proposed_schema(api_key, upload_id)
-    if schema_overrides is not None:
-        for field in schema_overrides:
-            schema["fields"][field] = schema_overrides[field]
-    if modality is not None:
-        schema["metadata"]["modality"] = modality
-    if id_column is not None:
-        schema["metadata"]["id_column"] = id_column
+
+    # if not simple zip upload, apply schema overrides
```

Comment:
can we add an error message for the case where a user does supply schema overrides to a simple zip upload?